firmware/kingst-la/make-readme.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+groff -Tascii -man sigrok-fwextract-kingst-la2016.1 | col -bx > README
+cat README
+
```

Comment:
I'd say the README generation should not only be a separate patch but it should also be done for all .1-files in firmware/. So, it will be consistend. Also, I think the script should be a Makefile (one directory above, of course) probably?

Reply:
Fair comment but my makefile foo is very weak :-(
Maybe someone else can help with this.

Reply:
Then I suggest to drop the README creation from this patch. We can discuss if we want READMEs separately. If the conclusion is we want them, then I can create the Makefile for it.